src/views/landing.html

```diff
@@ -69,6 +69,10 @@ <h2 class="govuk-heading-l">You provide trusted and authoritative local data</h2
       planning database software.
     </p>
 
+    <p class="govuk-body">
+      <a href="https://submit.planning.data.gov.uk/guidance" class="govuk-link">Read the guidance on how to provide planning and housing data.</a>.
```

Comment:
```suggestion
      <a href="/guidance" class="govuk-link">Read the guidance on how to provide planning and housing data.</a>.
```

Might be better to just do this - so it goes to the relative path rather than production url